.github/ISSUE_TEMPLATE/BUG-FORM.yml

```diff
@@ -0,0 +1,50 @@
+name: Bug report
+description: File a bug report
+labels: ["T-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please ensure that the bug has not already been filed in the issue tracker.
+
+        Thanks for taking the time to report this bug!
+  - type: dropdown
+    attributes:
+      label: Component
+      description: What component is the bug in?
+      multiple: true
+      options:
+        - Forge
+        - Cast
+        - Foundryup
+        - Other (please describe)
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Have you ensured that all of these are up to date?
+      options:
+        - label: Foundry
+        - label: Foundryup
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: What command(s) is the bug in?
+      description: Leave empty if not relevant
+      placeholder: "For example: forge test"
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: What operating system are you on?
+      options:
+        - Windows
+        - macOS (amd)
+        - macOS (M1)
+        - Linux
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: Please include relevant Solidity snippets as well if relevant.
```

Comment:
should also add output of `forge --version`

Reply:
Added! :smile: